# HG changeset patch
# User Rik <rik@octave.org>
# Date 1538063769 25200
#      Thu Sep 27 08:56:09 2018 -0700
# Node ID 06c41d1ae5b1d8bfc36823f5e5babaac708a1f9f
# Parent  c3503d044340beede0756ca06bb0833556d33def
Allow 'characters' as a value for the Units property of root object (bug #54742).

* graphics.in.h: Add "characters" to radio_property units of root object.

diff --git a/libinterp/corefcn/graphics.in.h b/libinterp/corefcn/graphics.in.h
--- a/libinterp/corefcn/graphics.in.h
+++ b/libinterp/corefcn/graphics.in.h
@@ -2917,17 +2917,17 @@ public:
       string_property fixedwidthfontname , "Courier"
       array_property monitorpositions r , default_screensize ()
       array_property pointerlocation , Matrix (1, 2, 0)
       double_property pointerwindow r , 0.0
       double_property screendepth r , default_screendepth ()
       double_property screenpixelsperinch r , default_screenpixelsperinch ()
       array_property screensize r , default_screensize ()
       bool_property showhiddenhandles , "off"
-      radio_property units U , "inches|centimeters|normalized|points|{pixels}"
+      radio_property units U , "{pixels}|inches|centimeters|points|normalized|characters"
       // Hide base properties which don't make sense for root object
       //radio_property beingdeleted h , "{off}|on"
     END_PROPERTIES
 
   private:
     std::list<graphics_handle> cbo_stack;
 
   };
